<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DesignScheduleWidget</class>
 <widget class="QWidget" name="DesignScheduleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QSpinBox" name="dayCountOfWeekSpinBox">
     <property name="prefix">
      <string>每周上课天数 - </string>
     </property>
     <property name="minimum">
      <number>5</number>
     </property>
     <property name="maximum">
      <number>7</number>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>保存</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="courseCountOfDaySpinBox">
     <property name="prefix">
      <string>每天课程数 - </string>
     </property>
     <property name="minimum">
      <number>4</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="value">
      <number>6</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QWidget" name="scheduleWidget" native="true"/>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
